pkg/logger/configure.go

```diff
@@ -14,7 +24,7 @@ import (
 var Version = "0.0.0"
 
 // ConfigureDevelopmentLogger configures a development logger which is more human readable instead of JSON
-func ConfigureDevelopmentLogger(level string, syncs ...io.Writer) (Logger, error) {
+func ConfigureDevelopmentLogger(ctx context.Context, level string, syncs ...io.Writer) (context.Context, error) {
```

Comment:
if you pass in a pointer to the ctx object you can add stuff to it

Reply:
Is there a reason you would prefer this pattern? Most things that I have read online that do context passing always wrap the context in a new value to preserve the values of the old context, should it be passed somewhere else (so it doesn't get mutated with the new span etc)

Reply:
ah fair enough if this is a standard pattern.